scargo/cli.py

```diff
@@ -52,7 +52,7 @@ def build(profile: str = Option("Debug", "--profile")) -> None:
 @cli.command()
 def check(  # pylint: disable=too-many-arguments
     clang_format: bool = Option(False, "--clang-format", help="Run clang-format."),
-    clang_tidy: bool = Option(False, "--clang-tidy", help="Run clang-tidy."),
+    # clang_tidy: bool = Option(False, "--clang-tidy", help="Run clang-tidy."),
```

Comment:
clang-tidy is also used in gitlab-ci.yaml pipeline file generated for the project. I will try to rather avoid disabling this option from the scope